components/ExerciseForm/ExerciseForm.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {KeyboardTypeOptions, ScrollView, View} from 'react-native';
+import {exercises} from '../../utils/constants';
+import {
+  ButtonType,
+  ExerciseDetailsForm,
+  ExerciseFormProps,
+  Fields,
+  FormField,
+} from '../../types/CommonTypes';
+import Form from '../Form/Form';
+
+import globalStyle from '../../assets/styles/globalStyle';
+
+const ExerciseForm: React.FC<ExerciseFormProps> = ({
+  submitButtonTitle,
+  onPressSubmit,
+  initialExerciseValues,
+}) => {
+  const type: ButtonType = 'submit';
+  const keyboardType: KeyboardTypeOptions = 'numeric';
```

Comment:
I've seen `type` and `keyboardType` variables in multiple files, maybe it's worth exporting these into `enum` or `object` for reusability purposes.